src/settings.ts

```diff
@@ -268,6 +269,8 @@ export function load(): ISettings {
             configuration.get<IStartAsLoginShellSettings>("startAsLoginShell", defaultStartAsLoginShellSettings),
         cwd: // NOTE: This must be validated at startup via `validateCwdSetting()`. There's probably a better way to do this.
             configuration.get<string>("cwd", undefined),
+        enableReferencesCodeLens:
+            configuration.get<boolean>("enableReferencesCodeLens", false),
```

Comment:
```suggestion
            configuration.get<boolean>("enableReferencesCodeLens", true),
```

Also change this one for consistency (yes, it's really annoying we have this in two, sometimes more places).